### ISSUE TITLE
ERROR sync: Panic occurred error=bitlists are different lengths topic=/eth2/beacon_attestation/ssz

### ISSUE THREAD

**Issue by ethers** [prysmaticlabs/prysm#4120]
```
[2019-11-26 22:34:22] ERROR sync: Panic occurred error=bitlists are different lengths topic=/eth2/beacon_attestation/ssz
goroutine 40290 [running]:
runtime/debug.Stack(0x0, 0x0, 0x0)
    GOROOT/src/runtime/debug/stack.go:24 +0x9d
runtime/debug.PrintStack()
    GOROOT/src/runtime/debug/stack.go:16 +0x22
github.com/prysmaticlabs/prysm/beacon-chain/sync.(*RegularSync).subscribe.func1.1(0xc001604200, 0xc0002de620)
    beacon-chain/sync/subscriber.go:124 +0x1e9
panic(0x1299560, 0x184d7c0)
    GOROOT/src/runtime/panic.go:679 +0x1b2
github.com/prysmaticlabs/go-bitfield.Bitlist.Or(0xc001bcb178, 0x2, 0x2, 0xc000543360, 0x3, 0x3, 0xc000bd97b0, 0x410ccb, 0xc000bd9868)
    external/com_github_prysmaticlabs_go_bitfield/bitlist.go:178 +0x11d
github.com/prysmaticlabs/prysm/proto/beacon/db.(*AttestationContainer).Contains(0xc001f01480, 0xc002066310, 0xc000bd9868)
    proto/beacon/db/attestation_container_helper.go:31 +0xef
github.com/prysmaticlabs/prysm/beacon-chain/operations.(*Service).HandleAttestation(0xc0003cb580, 0x18995c0, 0xc001b99bf0, 0x188fc00, 0xc002066310, 0x0, 0x0)
    beacon-chain/operations/attestation.go:189 +0x283
github.com/prysmaticlabs/prysm/beacon-chain/sync.(*RegularSync).beaconAttestationSubscriber(0xc0009a6c60, 0x18995c0, 0xc001b99bf0, 0x188fc00, 0xc002066310, 0x0, 0x0)
    beacon-chain/sync/subscriber_beacon_attestation.go:13 +0x6b
github.com/prysmaticlabs/prysm/beacon-chain/sync.(*RegularSync).subscribe.func1(0xc0009064b0, 0xec, 0xf0, 0xc0013aa800)
    beacon-chain/sync/subscriber.go:161 +0x89a
created by github.com/prysmaticlabs/prysm/beacon-chain/sync.(*RegularSync).subscribe.func2
    beacon-chain/sync/subscriber.go:187 +0x296
```

**Comment by rauljordan** [comment]
Is this resolved @prestonvanloon?

**Comment by prestonvanloon** [comment]
Fixed in #4209
